recipes/Xform/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "xforms" %}
+{% set version = "0.9999.3" %}
+
+package:
+   name: "{{ name|lower }}"
+   version: "{{ version }}"
+
+source:
+  url: http://software.igwn.org/sources/source/Xform-{{ version }}.tar.gz
+  sha256: 2752bec3cbf2c5e22472816999be35a45978436498dd01fdcec86d38ab4ade5d
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}     # [linux]
+    - {{ cdt('expat') }}     # [linux]
+    - meson >=0.50
+    - ninja
+    - pkg-config
+    - jpeg
+    - xorg-libx11
+  host:
+    - jpeg
+    - xorg-libx11
+    - xorg-libxfixes  # [linux]
+  run:
+    - jpeg
+    - xorg-libx11
+    - xorg-libxfixes  # [linux]
+
+test:
+  requires:
+    - pkg-config
+  commands:
+    - test -x ${PREFIX}/bin/fdesign  # [unix]
+    - test -f ${PREFIX}/lib/libfds.a  # [unix]
+    - test -f ${PREFIX}/lib/libforms.a  # [unix]
```

Comment:
We prefer static linkage on conda-forge, can you switch this to producing `so`s?

Reply:
@xhochy, that makes sense. Okay, I have switched this to producing `.so`'s.